moonstreamapi/moonstreamapi/routes/queries.py

```diff
@@ -456,17 +480,18 @@ async def update_query_data_handler(
 
         if "ext:csv" in tags:
             file_type = "csv"
+
+        json_payload["query"] = content
+        json_payload["params"] = request_update.params
+        json_payload["file_type"] = file_type
+        json_payload["blockchain"] = (
+            request_update.blockchain if request_update.blockchain else None

```

Comment:
if `request_update.blockchain is not None` but I think it could be just `= request_update.blockchain` without if condition